doc/src/arch/reference.rst

```diff
@@ -2401,6 +2401,12 @@ The full format is documented below.
 
         .. note:: See ``from_switchpoint_order`` for value descritpions.
 
+    :opt_param switch_override:
+
+        Specifies the name of a switch to be used to override the wire_switch of the segments in the ``to`` set. Can be used to create more complicated wire structures such as T and L shaped segments.
+        
+        **Default:** If no override is specified, the normal interconnect will be used. 
```

Comment:
I'd change "the normal interconnect will be used" to "the usual wire_switch that drives the ``to`` wire" will be used.